frontend/src/store.js

```diff
@@ -37,7 +37,7 @@ const rootReducer = combineReducers({
 
 // migration schema used to translate structure - return undefined to reset initialState
 const migrations = {
-  0: () => undefined,
```

Comment:
We should keep this back in

Reply:
Both! Ideally u should append new key-value pairing to the object rather than overriding it? If thAt makes sense haha

Reply:
ah icic thanks for picking this up 😃